### PR TITLE
Add point value import page

### PR DESCRIPTION
**Pull request by MertCingoz**

### CODE REVIEW

File: UI/resources/i18n.properties [hunk 1]
```diff
@@ -160,6 +160,7 @@ ui.dox.components=Components
 ui.dox.filters=Filters
 ui.dox.services=Services
 ui.dox.freeMarker=Free Marker Templates
+ui.dox.pointValueImport= Point value import
```
Comment (jazdw) [inline UI/resources/i18n.properties:163]: Extra space

File: UI/web-src/ngMango/components/pointValueImport/pointValueImport.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+        this.PointValueModification.import(file, params).then(
+            (data) => {
+                this.infoMessages = data;
+                this.DialogHelper.toastOptions({
+                    textTr: ['ui.app.pointValueImport.imported'],
+                    hideDelay: 5000
+                });
+            },
+            (err) => {
+                this.error = (err.data && err.data.cause) || err.mangoStatusText;
```
Comment (jazdw) [inline UI/web-src/ngMango/components/pointValueImport/pointValueImport.js:74]: Cause is only available to superadmin, just use mangoStatusText

File: UI/web-src/ngMango/services/pointValueModification.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+    static get $inject() {
+        return ['$http'];
+    }
+
+    constructor($http) {
+        const baseUrl = '/rest/latest/point-value-modification';
+
+        class pointValueModification {
+            static import(file, params) {
```
Comment (jazdw) [inline UI/web-src/ngMango/services/pointValueModification.js:21]: Can you move this to the maPointValues service

**Review by jazdw — CHANGES_REQUESTED** [review]
Only some small changes, all in all it looks very good. The drop area does not seem to work on Chrome, what browser did you test in?

**Comment by jazdw** [comment]
Including screenshots for @terrypacker @Joelhaggar to review

Download CSV from watchlist page (already existed, including for reference)
image

Link on admin home -
image

Page appearance -
image

Feedback (error is legitimate) -
image

**Comment by MertCingoz** [comment]
> Only some small changes, all in all it looks very good. The drop area does not seem to work on Chrome, what browser did you test in?

I am testing with Chrome and Firefox , both works with drop area for me
Chrome version 88.0.4324.190 (Official Build) (64-bit)
Firefox version 86.0 (64 bit)

**Comment by jazdw** [comment]
Looks like .csv files have a different MIME type on my machine since I have Excel installed
image

Maybe just check the file extension only?

**Comment by MertCingoz** [comment]
I have found possible mime types for a csv file and changed the code to accept these

```
  'application/csv',
  'application/x-csv',
  'application/vnd.ms-excel',
  'text/csv',
  'text/x-csv',
  'text/comma-separated-values',
  'text/x-comma-separated-values'
```

**Pull request merged**